docs/plugins.md

```diff
@@ -87,7 +87,7 @@ These plugins apply transformations to your code.
 - [es2015-block-scoping](/docs/plugins/transform-es2015-block-scoping/)
 - [es2015-classes](/docs/plugins/transform-es2015-classes/)
 - [es2015-computed-properties](/docs/plugins/transform-es2015-computed-properties/)
-- [es2015-constants](/docs/plugins/check-es2015-constants/)
```

Comment:
ah ...I see. Then my change doesn't make sense of course. I was confused to find a none transform plugin under the `transfomer` section and automatically assumed this is wrong.

I will close the PR.
Thanks!